SharePointDsc/Modules/SharePointDsc.Farm/SPFarm.psm1

```diff
@@ -348,3 +348,60 @@ function Remove-SPDscConfigDBLock
         }
     }
 }
+
+<#
+
+.SYNOPSIS
+
+Get-SPDscConfigDBConnectionEncryption returns the current Value for the DatabaseConnectionEncryption and DatabaseServerCertificateHostName.
+
+.DESCRIPTION
+
+Get-SPDscConfigDBConnectionEncryption returns the current Value for the DatabaseConnectionEncryption and DatabaseServerCertificateHostName.
+
+.EXAMPLE
+
+Get-SPDscConfigDBConnectionEncryption
+
+#>
+function Get-SPDscConfigDBConnectionEncryption
+{
+    param
+    (
+    )
+
+    Write-Verbose -Message "Removing lock for database $Database"
```

Comment:
This verbose message needs updating. This function doesn't handle the locking but retrieves the current settings of the encryption state. The verbose message should state that.